ui/raidboss/data/07-dt/dungeon/origenics.ts

```diff
@@ -656,7 +676,7 @@ const triggerSet: TriggerSet<Data> = {
     },
     {
       'locale': 'ja',
-      'missingTranslations': true,
+      'missingTranslations': false,
       'replaceSync': {
```

Comment:
```suggestion
      'locale': 'ja',
      'replaceSync': {
```
This repository prefers to delete this line rather than setting `'missingTranslations': false`.